lib/compeon/access_token.rb

```diff
@@ -33,6 +37,8 @@ def parse(token)
         client_id = data.fetch('cid')
 
         new(role: role, user_id: user_id, kind: kind, client_id: client_id, token: token)
+      rescue JWT::DecodeError
+        raise ParseError
```

Comment:
```suggestion
        raise ParseError, e
```
This will change the error message from 
```
ParseError: ParseError
```
to
```
ParseError: Not enough or too many segments
```